mkdocs/themes/mkdocs/css/base.css

```
@@ -42,32 +44,41 @@ body > .container {
     display: inline-block;
     padding: 4px;
     line-height: 1.428571429;
-    background-color: #fff;
-    border: 1px solid #ddd;
+    /* csslint ignore:start */
+    background-color: var(--bs-secondary-bg-subtle);
+    border: 1px solid var(--bs-secondary-border-subtle);
+    /* csslint ignore:end */
     border-radius: 4px;
     margin: 20px auto 30px auto;
 }
 
 h1 {
-    color: #444;
+    /* csslint ignore:start */
+    color: var(--bs-secondary-text-emphasis);
```

I think this color is not appropriate, it's really grey. On Bootstrap's own site, headings are just the default black color :thinking: 

Based on this page I was confused
https://getbootstrap.com/docs/5.3/customize/css-variables/
It says `  --bs-heading-color: inherit;` but .. that's not the actual value.
Maybe we just do `inherit` directly - that looks good.

```suggestion
    color: inherit;
```

---

I was just trying to maintain what was there using Bootstrap variables. The existing style defined a gray color so I used the closest variable color I could find. But if you want to make the change that is fine with me.

---

Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae